lib/fog/test_helpers/minitest/assertions.rb

```diff
@@ -0,0 +1,35 @@
+require "fog/schema/data_validator"
+
+# Format related hackery
+# allows both true.is_a?(Fog::Boolean) and false.is_a?(Fog::Boolean)
+# allows both nil.is_a?(Fog::Nullable::String) and ''.is_a?(Fog::Nullable::String)
+module Fog
```

Comment:
Yeah, consolidating vs duplicating sounds like a good plan to help reduce the likelihood of drift in the future. Thanks!